.eslintrc.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
 	},
 	rules: {
 		'no-unused-vars': 'off',
+		'@typescript-eslint/no-explicit-any': 'off',
```

Comment:
Do not disable `@typescript-eslint/no-explicit-any`. We should be using proper type/interface definitions or casts instead.

Reply:
Absolutely